{"pytest_version": "7.4.3", "$report_type": "SessionStart"}
{"nodeid": "", "outcome": "passed", "longrepr": null, "result": null, "sections": [], "$report_type": "CollectReport"}
{"nodeid": "tests27_89_1.py", "outcome": "passed", "longrepr": null, "result": null, "sections": [], "$report_type": "CollectReport"}
{"nodeid": "tests27_89_1.py::test_presence_of_inserted_element", "location": ["tests27_89_1.py", 4, "test_presence_of_inserted_element"], "keywords": {"test_presence_of_inserted_element": 1, "tests27_89_1.py": 1, "Folder_89": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00012328801676630974, "start": 1701648186.761105, "stop": 1701648186.761229, "$report_type": "TestReport"}
{"nodeid": "tests27_89_1.py::test_presence_of_inserted_element", "location": ["tests27_89_1.py", 4, "test_presence_of_inserted_element"], "keywords": {"test_presence_of_inserted_element": 1, "tests27_89_1.py": 1, "Folder_89": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/Benchmark_GPT3_5/Q27/gpt_3_5_turbo_results_1/Folder_89/generated_answer.py", "lineno": 4, "message": "ValueError: 62 is not in list"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_presence_of_inserted_element():", "        initial_list = [set() for i in range((62 + 1) * 2)]", ">       returned_list = insert_after_index(initial_list)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q27/gpt_3_5_turbo_results_1/Folder_89/tests27_89_1.py", "lineno": 7, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def insert_after_index(lst):", "        new_lst = lst.copy()", ">       new_lst.insert(new_lst.index(62) + 1, 51)", "E       ValueError: 62 is not in list"], "reprfuncargs": {"args": [["lst", "[set(), set(), set(), set(), set(), set(), ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q27/gpt_3_5_turbo_results_1/Folder_89/generated_answer.py", "lineno": 4, "message": "ValueError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_presence_of_inserted_element():", "        initial_list = [set() for i in range((62 + 1) * 2)]", ">       returned_list = insert_after_index(initial_list)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q27/gpt_3_5_turbo_results_1/Folder_89/tests27_89_1.py", "lineno": 7, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def insert_after_index(lst):", "        new_lst = lst.copy()", ">       new_lst.insert(new_lst.index(62) + 1, 51)", "E       ValueError: 62 is not in list"], "reprfuncargs": {"args": [["lst", "[set(), set(), set(), set(), set(), set(), ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q27/gpt_3_5_turbo_results_1/Folder_89/generated_answer.py", "lineno": 4, "message": "ValueError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/Benchmark_GPT3_5/Q27/gpt_3_5_turbo_results_1/Folder_89/generated_answer.py", "lineno": 4, "message": "ValueError: 62 is not in list"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00011387397535145283, "start": 1701648186.761342, "stop": 1701648186.761456, "extras": [], "$report_type": "TestReport"}
{"nodeid": "tests27_89_1.py::test_presence_of_inserted_element", "location": ["tests27_89_1.py", 4, "test_presence_of_inserted_element"], "keywords": {"test_presence_of_inserted_element": 1, "tests27_89_1.py": 1, "Folder_89": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.00013048597611486912, "start": 1701648186.764887, "stop": 1701648186.765019, "$report_type": "TestReport"}
{"exitstatus": 1, "$report_type": "SessionFinish"}
